index.css

```diff
@@ -1,466 +1,709 @@
+  max-width: 640px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: var(--text-color);
+}
+
+.test4 {
+  text-align: justify;
+  margin-bottom: 20px;
+}
+
+.about {
+  margin-top: 90px;
+  border-radius: 0 4em 0 0;
+  background-size: cover;
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 24px;
+  left: calc(50% - 375px / 2);
+  background-color: #fff;
+}
+
+.about-icons {
+  color: var(--icon-color);
+  font-size: 20px;
+  padding-top: 30px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 50%;
+  margin-bottom: 30px;
+}
+
+.fa-angle-down {
+  display: flex;
+  justify-content: space-between;
+  color: black;
+}
+
+.fa-angle-right {
+  display: flex;
+  justify-content: space-between;
+  color: black;
+}
+
+.about-heading-small {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.about-skills-container {
+  flex-direction: column;
+  padding: 2em 1em;
+  width: 100%;
+}
+
+.about-skills-container > hr {
+  height: 2px;
+  background: #dfe1e6;
+  border: none;
+}
+
+.about-skill-heading {
+  justify-content: space-between;
+  padding: 2em 1em 2em 0;
+}
+
+.about-skill-body {
+  align-items: center;
+  justify-content: flex-start;
+  background-color: #f7f7f9;
+  border-radius: 0.5em;
+  margin: 0.5em 0;
+  padding: 0.5em;
+}
+
+.about-skill-text {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  padding-left: 1em;
+  line-height: 20px;
+  color: #253858;
+}
+
+.cont {
+  background-color: white;
+}
+
+footer {
+  padding-top: 40%;
+  padding-bottom: 40%;
+  width: 375px;
+  height: 779px;
+  background-image: url(./assets/footerbg.png);
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+footer span {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+footer p {
+  width: 90%;
+  margin-bottom: 3%;
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+}
+
+footer form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 100%;
+  padding: 5%;
+  gap: 1em;
+}
+
+footer form .text-inputs {
+  border-radius: 8px;
+  border-color: #cfd8dc;
+  line-height: 2em;
+  width: 327px;
+  font-size: large;
+  padding: 5px;
+}
+
+footer form .message-input {
+  height: 170px;
+  border-radius: 8px;
+  border-color: #cfd8dc;
+  width: 327px;
+  font-size: large;
+  padding: 5px;
+  background-color: #fff;
+  left: 16px;
+  right: 38px;
+  top: 15px;
+  bottom: 15px;
+}
+
+@media (min-width: 768px) {
```

Comment:
Good job with following mobile first approach. But as you can see below there are a few issues when it is extending the view from mobile to desktop. Mostly on the right side. Kindly fix them to have a full responsive portfolio 👍 
![Screen Recording 2023-02-25 at 9 03 38 PM](https://user-images.githubusercontent.com/25887253/221373677-ebeb3692-b5e7-4d38-b8b8-1e1607f1e918.gif)